source/includes/_custom_screeners.md

```diff
@@ -0,0 +1,197 @@
+```
+
+### HTTP Request
+
+`POST https://www.tapresearch.com/api/v1/custom_qualifications`
+
+### Required Parameters
+Parameter | Type | Description
+--------- | ---- | -----------
+question_text | String | This is the question text that will be served to our respondents.
+answer_type | Integer | This value will determine how campaign_qualifications pre-codes are generated for quotas. See **Answer Types** section for more information.
+country_language_id | Integer | Country language targeting for the qualification.
+
+### Optional Parameters
+Parameter | Type | Description
+--------- | ---- | -----------
+name | String | Name of the custom qualification.
+country | String | Country that is being targeted.
+country_id | Integer | Country specific ID used for internal purposes.
+customer_id | Integer | The unique identifier associated with your account. See **Callbacks** section for more information.
+
+### Response
+Parameter | Type | Description
+--------- | ---- | -----------
+name | String | Name of the custom qualification.
+question_id | Integer | This value will be used to fetch qualification details.
+question_text | String | This is the question text that will be served to our respondents.
+answer_type | Integer | This value will determine how campaign_qualifications pre-codes are generated for quotas. See **Answer Types** section for more information.
+country_language_id | Integer | Country language targeting for the qualification.
+en_translation | String | The English translation of the question.
+
+## Create a custom qualification answer
+
+Create a custom qualification answer to correspond with your custom qualifications.
+
+> Sample Request Payload - Update custom qualification name
+
+```json
+{
+    "option_text": "Corporate Chef"
+}
+```
+
+> Sample Response 
+
+```json
+{
+    "option_text": "Corporate Chef",
+    "pre_code": 4,
+    "en_translation": "Corporate Chef",
+    "country_language_id": 1
+}
+```
+
+### HTTP Request
+
+`POST https://www.tapresearch.com/api/v1/custom_qualifications/:question_id/custom_qualification_answers`
+
+
+### Required Parameters
+Parameter | Type | Description
+--------- | ---- | -----------
+option_text | String | The answer text that is shown to our respondents.
+
+### Response
+Parameter | Type | Description
+--------- | ---- | -----------
+option_text | String | The answer text that is shown to our respondents.
+pre_code | Integer | This is the value(s) you will be passing into the campaign_qualifications pre_codes array to describe which respondents qualify for a survey.
+en_translation | String | The English translation of the question.
+country_language_id | Integer | Country language targeting for the qualification.
+
+## Create a campaign qualification
+
+Create a campaign qualification which will allow you to associate your custom qualifications and answers to a specific campaign.
+
+> Sample Request Payload - 
+
+```json
+{
+    "question_id": 100112713,
+    "pre_code_values": "1"
```

Comment:
This is referred to as an Int in line 100 but then is a string here.  Lets use string everywhere for pre_code_values

Reply:
All references to pre_code_values have been changed to string.